.editorconfig

```diff
@@ -2,13 +2,12 @@ root = true
 
 [*]
 charset = utf-8
-indent_size = 2
+indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.{kt,kts}]
-indent_size = 4
```

Comment:
I suggest we can unify all files to indent_size 2, to keep code clean and less nesting.

Reply:
As for me XML with indent size 2 looks strange. Google uses 4 as default

Reply:
You can leave them as size 4 explicitly. I would prefer the global size 2.

Reply:
Ok, will leave default as 2 and make 4 for xml

Reply:
#140

Reply:
b9dbf75c7e437e70e74c0a2a4ab7713979aa5254